1.purpose_and_goals.md

```diff
@@ -1,24 +1,12 @@
 # 1. Purpose and Goals
 
-The Open Application Model defines a standard, platform-agnostic way to describe cloud and edge applications. The goal of the specification is to provide a common way to model application deployments agnostic to any specific container runtime, orchestration software, cloud provider, or hardware configuration, with clearly defined separation of concerns, while still allowing implementations to make use of the native APIs, tools, and features that are unique to the implementation and its underlying platform.
+The goal of Open Application Model is to define a standard, runtime-agnostic way to describe application deployment across hybrid environments, clouds or even edge devices.
```

Comment:
What's the difference between runtime and platform?

Reply:
No big difference, this is a consistency fix. Though in my mind, "runtime" specifically describes where the app is running.